nntrainer/tensor/tensor_pool.cpp

```diff
@@ -131,8 +143,6 @@ void TensorPool::finalize(const MemoryPlanner &planner,
     /** 2. for each tensor request if it is in the provided range */
     if (validity_end < start_order || validity_start > end_order)
       continue;
```

Comment:
Shouldn't this always be false (so never continue) because validity_end = validity_start is always >= start_order.

Could you check if initial value of `validity_end` = `start_order - 1` ?

Reply:
this is a sanity check in case the previous calculations does do anything wrong.

Reply:
I mean sanity check will always pass because validity_end < start_order will be always false, if this is intended, I think this line should be removed instead.

Reply:
validity_end can be less than start_order, when the full range of validity is less than the start_order itself.

Reply:
Resolved offline that `validity_end < start_order` remains always false but will be here for completeness.